index.bs

```diff
@@ -127,69 +101,54 @@ The Accelerometer Interface {#accelerometer-interface}
+interface represents the <a>acceleration</a> along Y-axis.
+This attribute returns [=map/value=] for [=latest reading=]["y"] [=map/entry=].
 
-<pre class="idl">
-  [Constructor(AccelerometerReadingInit AccelerometerReadingInit)]
-  interface AccelerometerReading : SensorReading {
-      readonly attribute double x;
-      readonly attribute double y;
-      readonly attribute double z;
-  };
+### Accelerometer.z ### {#accelerometer-z}
 
-  dictionary AccelerometerReadingInit {
-      double x = 0;
-      double y = 0;
-      double z = 0;
-  };
-</pre>
+The {{Accelerometer/z!!attribute}} attribute of the {{Accelerometer}}
+interface represents the <a>acceleration</a> along Z-axis.
+This attribute returns [=map/value=] for [=latest reading=]["z"] [=map/entry=].
 
-### The Accelerometer attributes ### {#accelerometer-attributes}
+### Accelerometer.includesGravity ### {#accelerometer-includesGravity}
 
-The <a attribute for="Accelerometer">includesGravity</a> attribute of the {{Accelerometer}}
-interface represents whether the acceleration information provided by the sensor includes effect of the gravity
-force. In case when <a attribute for="Accelerometer">includesGravity</a> equals to false, {{Accelerometer}}
+The {{Accelerometer/includesGravity!!attribute}} attribute of the {{Accelerometer}}
+interface represents whether the <a>acceleration</a> information provided by the sensor includes the effect of the gravity
+force. In case when {{Accelerometer/includesGravity!!attribute}} equals to false, {{Accelerometer}}
 will provide <a>linear acceleration</a> information.
 
-### The AccelerometerReading constructor ### {#accelerometer-reading-constructor}
-
-The AccelerometerReading constructor accepts {{AccelerometerReadingInit}} dictionary that is used
-for initialization of {{AccelerometerReading}} attributes.
-
-### The AccelerometerReading attributes ### {#accelerometer-reading-attributes}
-
-The <a attribute for="AccelerometerReading">x</a> attribute of the {{AccelerometerReading}}
-interface represents the <a>acceleration</a> along X-axis.
+The AccelerometerOptions Dictionary {#accelerometer-options-dictionary}
+--------------------------------
 
-The <a attribute for="AccelerometerReading">y</a> attribute of the {{AccelerometerReading}}
-interface represents the <a>acceleration</a> along Y-axis.
+<pre class="idl">
+  dictionary AccelerometerOptions :  SensorOptions  {
+    boolean includeGravity = true;
+  };
+</pre>
 
-The <a attribute for="AccelerometerReading">z</a> attribute of the {{AccelerometerReading}}
-interface represents the <a>acceleration</a> along Z-axis.
+By default, accelerometer sensor provides <a>acceleration</a> information including the effect of the gravity force.
+In cases when <a>linear acceleration</a> information is required, {{AccelerometerOptions}} dictionary
+must include an [=map/entry=] with a [=map/key=] that is "includesGravity" and its [=map/value=] must be a
```

Comment:
A more concise form would be:

[=map/entry=] whose [=map/key=] is "includesGravity" and whose value is false.

"Whose" is commonly used language in specs, type is defined in IDL so can drop that from prose.